install/install.sh

```diff
@@ -69,6 +69,13 @@ echo "Going to install companion-docker version ${VERSION}."
 echo "Downloading and installing udev rules."
 curl -fsSL $ROOT/install/udev/100.autopilot.rules -o /etc/udev/rules.d/100.autopilot.rules
 
+echo "Download default Ardusub firmwares for Navigator and Pixhawk1."
```

Comment:
I don't think that this is the best place to install it, since is totally not relevant for the installation step and is only specific for the ardupilot manager, can't we have it in the setup.py as we do for the js files ?

Reply:
Makes sense. Will transfer.